setup.py

```diff
@@ -22,11 +23,11 @@
 
 setup(
     name='pubsub-logging',
-    version='0.1.6',
+    version='0.2.0',
     author='Takashi Matsuo',
     author_email='tmatsuo@google.com',
     url='https://github.com/GoogleCloudPlatform/cloud-pubsub-logging-python',
-    py_modules=['pubsub_logging'],
+    packages=['pubsub_logging'],
     description="A logging handler for publishing the logs to Cloud Pub/Sub",
```

Comment:
A logging handler for publishing log messages to Cloud Pub/Sub

A handler for logging to Cloud Pub/Sub